lib/fileutils.rb

```diff
@@ -1018,14 +1018,15 @@ def copy_stream(src, dest)
   end
   module_function :copy_stream
 
-  # Moves files from +src+ (a single path or an array of paths)
-  # to +dest+ (a single path).
-  # If +src+ and +dest+ are on different devices,
-  # first copies, then removes +src+.
+  # Moves entries.
   #
-  # Arguments +src+ and +dest+
+  # Arguments +src+ (a single path or an array of paths)
+  # and +dest+ (a single path)
   # should be {interpretable as paths}[rdoc-ref:FileUtils@Path+Arguments].
   #
+  # If +src+ and +dest+ are on different devices,
+  # first copies, then removes +src+.
```

Comment:
I think the definition of "device" is confusing and ambiguous. I think of one drive as a "device", but a drive can have multiple file systems.

The man page for `mv` says the following:

> As the rename(2) call does not work across file systems, mv uses cp(1) and rm(1)
> to accomplish the move.

```suggestion
  # If +src+ and +dest+ are on different file systems,
  # first copies, then removes +src+.
```

Reply:
Fixed (three places).